stacks/java-quarkus/stack.yaml

```diff
@@ -5,4 +5,5 @@ icon: https://design.jboss.org/quarkus/logo/final/SVG/quarkus_icon_rgb_default.s
 versions:
   - version: 1.3.0
   - version: 1.4.0
+  - version: 1.5.0
```

Comment:
why do we need 1.5.0 version? can we simply update 1.4.0 if it is completely broken?
I'm not a huge fan of versioning in general. Can we have only single version?

Reply:
> can we simply update 1.4.0 if it is completely broken?

It is broken not because of the devfile but because maven compiler was updated in the project's `pom.xml`.
> Can we have only single version?

This is a question to @maxandersen

Reply:
I'm not sure what the version is used for?

The original intention was it would create a latest greatest supported config. 

If need to control java version used there is a parameter for that when downloading the zip. 

Would that make it possible to have less versions ?

Reply:
@maxandersen 
> If need to control java version used there is a parameter for that when downloading the zip.

Could you please elaborate more on this? How can I pass the java version?

Reply:
I have figured out how to pass the java version. Closing this PR and preparing a new one.

Reply:
https://github.com/devfile/registry/pull/366